docs/shared/guides/module-federation/micro-frontend-architecture.md

```diff
@@ -1,9 +1,13 @@
 # Micro Frontend Architecture
 
-Since version 14, Nx provides out-of-the-box [Module Federation](/concepts/more-concepts/faster-builds-with-module-federation) support to both React
-and Angular. The Micro Frontend (MFE) architecture builds on top of Module Federation by providing _independent deployability_.
-
-If you have not read the [Module Federation guide](/concepts/more-concepts/faster-builds-with-module-federation) yet, we recommend that you read it
+Since version 14, Nx provides
```

Comment:
Content seems the same is this a formatting issue?

Reply:
Yeah, my ide ran code formatting over the md file.